workshops/maze/README.md

```diff
@@ -0,0 +1,113 @@
+scratch, we give you the code for how to make the maze game show above and
+your mission will be to make your own version of it.
+
+This workshop also, assumes that you have done the
+[Dodge](https://github.com/hackedu/hackedu/blob/master/workshops/dodge/README.md)
+workshop. Make sure you do that workshop first.
+
+## 3. Setup
+
+1. If you're more experienced and want to use your own editor, go ahead. Use the
+   below files to your editor. Do note however that the rest of this tutorial be
+   written assuming that you are using JS Bin.
+
+    - [this HTML file](https://gist.githubusercontent.com/jonleung/11ecd13a9d957003416b/raw/625081d81cc9c941007ddfd54c9ffdeec0782d62/index.html)
+    - [this JavaScript file](https://gist.githubusercontent.com/jonleung/11ecd13a9d957003416b/raw/625081d81cc9c941007ddfd54c9ffdeec0782d62/main.js)
+    - and [this CSS file](https://gist.githubusercontent.com/jonleung/11ecd13a9d957003416b/raw/625081d81cc9c941007ddfd54c9ffdeec0782d62/style.css)
+
+2. If you haven't already, make sure you have a JS Bin account first. <a
+   href="https://jsbin.com/register" target="_blank_">Create one here.</a>
+3. Then open <a target="_blank_"
+   href="#TODO"> this JS Bin</a>. It has
+   all the starter code we need for this tutorial. Write any new code in this JS
+   Bin. We'll refer to this JS Bin as your _"working bin"_.
+
+## 4. Understanding the Code
+
+Let's understand the code in your working bin. There are two main parts:
+
+1. Adding the maze sprite
+2. Removing the maze if the player touches the maze
+
+### Adding the Maze Sprite
+
+Here is the top half of the code that is responsible for adding the maze sprite:
+
+```js
+var maze;
+var mazeImg;
+
+function setup() {
+  createCanvas(400, 400);                 // create a canvas
+
+  maze = createSprite(200, 200);          // create a sprite in the center
+  mazeImg = loadImage('http://i.imgur.com/ImtI8zi.png');    // load an image
+  maze.addImage(mazeImg);                 // make the sprite have an image
+}
+
+function draw() {
+  background(255, 255, 255);              // clear the background
+  drawSprites();                          // draw all the sprites
+```
+
+For a refresher for how to do the following, see the corresponding
+sections from the Dodge workshop
+
+- [creating a blank canvas](https://github.com/hackedu/hackedu/blob/master/workshops/dodge/blank_canvas.md)
+- [adding a sprite](https://github.com/hackedu/hackedu/blob/master/workshops/dodge/add_player_sprite.md)
+- [adding an image to your sprite](https://github.com/hackedu/hackedu/blob/master/workshops/dodge/player_image.md)
+
+### Game Over Condition
+
+The remainder of the code is responsible for removing the maze if the the mouse
+touches it.
+
+```js
+  if(maze.overlapPixel(mouseX, mouseY)){  // if the mouse touches the image
+    maze.remove();                           // remove the maze
+  }
+}
+```
+
+Note that `mouseX` represents the current **x** coordinate of the mouse and
```

Comment:
@paked yes! I'll like to this: https://github.com/hackedu/hackedu/blob/master/workshops/soccer/add_player_sprite.md#understanding-the-coordinate-system